templates/base/base.html

```diff
@@ -7,37 +7,43 @@
         <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.1.3/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-1BmE4kWBq78iYhFldvKuhfTAU6auU8tT94WrHftjDbrCEXSU1oBoqyl2QvZ6jIW3" crossorigin="anonymous">
         <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap-icons@1.7.2/font/bootstrap-icons.css">
         <link rel="stylesheet" type="text/css" href="{{ url_for('static', filename='css/base/base.css') }}"/>
+        <link rel="icon" type="image/x-icon" href="{{ url_for('static', filename='imgs/favicon.ico') }}">
+        <link rel="preconnect" href="https://fonts.googleapis.com">
```

Comment:
why 4 the same sources??

Reply:
oh, writing code to 4am has some drawbacks apparently, will wipe this out XD